core/jobs/batch_jobs/story_migration_jobs_test.py

```diff
@@ -99,7 +106,10 @@ def setUp(self):
             'next_node_id': 'node_2222'
         }
         self.broken_contents = copy.deepcopy(self.latest_contents)
-        self.broken_contents['nodes'][0]['description'] = 123
+        # The expected type of `description` key is string but here for testing
+        # purposes we are providing a integer value, which causes MyPy to throw
+        # an error. Thus to silent the error, we used ignore here.
+        self.broken_contents['nodes'][0]['description'] = 123  # type: ignore[arg-type]
```

Comment:
Should a TODO be added here?

Reply:
I think you are right because in line 152 we are performing a test `broken_skill_is_not_migrated` due to wrong type of value.

Reply:
Comment is changed, Now TODO is added.